### DISCUSSION TITLE
emoji pickerの無効化について

### DISCUSSION THREAD

**Discussion by Monoalice** [growilabs/growi#7593] [Help / Questions] [answered]
長年大変お世話になっております。
4系から6系にバージョンアップしたところ、ページの編集で度々emoji pickerが誤発動してしまうようになりました。
※IPアドレス:ポート番号の入力で動作してしまう。。

emoji pickerを無効化する方法はないでしょうか。
よろしくお願いいたします。

**Comment by jam411** [comment]
related to #6834

**Comment by yuki-takei** [comment]
2023Q4 リリース予定の GROWI v7 で、CodeMirror の autocompletion 機能を利用した実装になる予定です。

**Comment by yuki-takei** [comment, marked as answer]
v7 で v4 系と同様の挙動になりました。
